### PR TITLE
Remove ScheduleUtil，use the existing CronUtils

### PR DESCRIPTION
**Pull request by lfyee**
## What is the purpose of the pull request
CronUtils.java is a public quartz tools, so delete the ScheduleUtil.java, then the project dolphinscheduler-api can remove dolphinscheduler-server

## Verify this pull request
the command 'mvn clean package -Dmaven.test.skip=true' is success

  - *remove ScheduleUtilsTest
  - *remove ScheduleUtils
  - *edit ExecutorService  for use CronUtils
  - *edit MasterExecThread  for use CronUtils

### CODE REVIEW

**Comment by Jave-Chen** [comment]
```
CronExpression cronExpression = null;
try {
    cronExpression = CronUtils.parse2CronExpression(item.getCrontab());
    List<Date> list = CronUtils.getSelfFireDateList(start, end, cronExpression);
    listDate.addAll(list);
} catch (ParseException e) {
    logger.error(e.getMessage(), e);
    continue;
}
```

These code are duplicated, is it possible to encapsulate then as a method?

**Comment by lfyee** [comment]
@Jave-Chen ,use public methods to simplify duplicate code

**Comment by Jave-Chen** [comment]
Add include record at incubator-dolphinscheduler/pom.xml will get right UT coverage.

File: dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/cron/CronUtilsTest.java [hunk 1]
```diff
@@ -180,4 +182,17 @@ public void test2(){
             }
         }
     }
+
+
+    @Test
+    public void test3(){
```
Comment (Technoboy-) [inline dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/cron/CronUtilsTest.java]: testGetSelfFireDateList

Comment (lfyee) [reply dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/cron/CronUtilsTest.java]: ok, it has been modified

**Comment by Jave-Chen** [comment]
+1  good job

File: dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/cron/CronUtilsTest.java [hunk 2]
```diff
@@ -68,9 +70,9 @@ public void testCronAsString() {
     @Test
     public void testCronParse() throws ParseException {
         String strCrontab = "0 1 2 3 * ? *";
-        strCrontab = "0/50 0/59 * * * ? *";
-        strCrontab = "3/5 * 0/5 * * ? *";
-        strCrontab = "1/5 3/5 1/5 3/30 * ? *";
+//        strCrontab = "0/50 0/59 * * * ? *";
+//        strCrontab = "3/5 * 0/5 * * ? *";
+//        strCrontab = "1/5 3/5 1/5 3/30 * ? *";
```
Comment (Jave-Chen) [inline dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/cron/CronUtilsTest.java]: I think 73-75 line can be removed.

Comment (lfyee) [reply dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/cron/CronUtilsTest.java]: ok, removed

**Pull request merged**